angularjs-portal-home/src/main/webapp/my-app/layout/services.js

```diff
@@ -38,6 +38,33 @@ define(['angular', 'jquery'], function(angular, $) {
                 },
             });
         };
+        
+        var addToLayoutByFname = function addToLayoutByFname(fname){
+          var tabName = SERVICE_LOC.layoutTab;
+          return $.ajax({
```

Comment:
:arrow_right: [`$http.post`](https://docs.angularjs.org/api/ng/service/$http#post) would be preferable over [`$.ajax`](https://api.jquery.com/jquery.ajax/)